pkgs/development/python-modules/pyobjc/default.nix

```diff
@@ -0,0 +1,19 @@
+{ fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "pyobjc";
+  version = "4.0b1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16bng6960c1m57nnh1l09ycnyimrqzw9mx9pnyjxn5zzm5kalr37";
+  };
+
+  meta = {
+    description = "A bridge between the Python and Objective-C programming languages";
+    license = stdenv.lib.licenses.mit;
```

Comment:
`undefined variable ‘stdenv’ at /nix/store/9nl1fs575ayk3hk1bjymp03plh84wnrp-quodlibet.tar.gz/pkgs/development/python-modules/pyobjc/default.nix:16:24`

Please do test.

Reply:
can't, don't have a mac :/

Reply:
actually I could've caught that... added licenses after building, whoops.

Reply:
That's irrelevant. I got this evaluation error on NixOS. `stdenv` is not passed in.

Reply:
Yea, its a common error. Happens to me as well.